extensions/pyRevitTools.extension/pyRevit.tab/Drawing Set.panel/views.stack/Views.pulldown/Move Viewport Label.pushbutton/script.py

```diff
@@ -0,0 +1,24 @@
+# encoding: utf-8
+# from https://discourse.pyrevitlabs.io/t/i-developed-a-tool-need-help-to-get-it-into-the-new-release-of-pyrevit/7639/3
+
+from pyrevit import DB, script, revit, HOST_APP
+
+doc = HOST_APP.doc
+uidoc = HOST_APP.uidoc
+script.get_output().close_others()
```

Comment:
As it is now, this command doesn't even use the output window, so it shouldn't take care of closing the other... it should be a global option/behavior not tied to a particular button